www/js/app.js

```diff
@@ -792,6 +804,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
     // Since late 2014, all ZIM files should use relative URLs
     var regexpImageUrl = /^(?:\.\.\/|\/)+(I\/.*)$/;
     var regexpMetadataUrl = /^(?:\.\.\/|\/)+(-\/.*)$/;
+    
+    // Cache of the CSS as blobs 
+    var cssBlobCache = new Map();
```

Comment:
Just one more question before I squash/merge, @Mossroy:
This is *not* technically a blob cache any more. It started off as holding blob URLs, but due to CSP, it's become just a CSS cache (the cache just stores the CSS filename and the raw CSS extracted from the ZIM). Would you prefer the function names and variable names to be changed to ...CssCache... instead of ...CssBlobCache... ? It's easy to do in my editor.

Reply:
Yes. Please do.